### ISSUE TITLE
hashicorp fixation with 1344px

### ISSUE THREAD

**Issue by infowolfe** [hashicorp/vault#27124]
@hashishaw @hellobontempo 
(sorry to tag y'all by name, but you both touched the file referenced below)

Could you please explain why Hashicorp sometimes refuses to acknowledge that people have monitors that have resolutions >=1409 px in width?

https://github.com/hashicorp/vault/blob/25f038afacc253cde77d610d0d77f66fe4cffaea/ui/app/styles/components/cluster-banners.scss#L8

**Describe the bug**
Like most people who use computers in 2024 I have a monitor that is >=1409px in width, and yet some sites across Hashicorp (such as the terraform registry) have a `max-width:1344px` set.

**To Reproduce**
1. Have a 1080p or 4K monitor (with hidpi off)
2. Maximize your window
3. Open something that uses `.cluster-banners-wrapper` css style or another container where `max-width` is set.
4. Note that whatever's in this container has a large amount of empty/unused screen space to either side of it, defeating the purpose of having a large monitor.

**Expected behavior**
`max-width` (at least on terraform registry's documentation) should either not be set, or set to a value that allows full utilization of the given viewport.

**Environment:**
any web browser

**Additional context**
I found this little gem because I had to use horizontal scrollbars when trying to review code on registry.terraform.io and this happens to be the only reference to the infernal 1344px I could find in hashicorp's public code on github.
 
tl;dr: please stop artificially limiting the max-width org-wide, thanks!

Offending code from terraform registry (which doesn't have a github project that I can file bugs against):
```
@media screen and (max-width: 1407px) {
    .container.is-fullhd:not(.is-max-desktop):not(.is-max-widescreen),.section-browse-list .is-fullhd.browse-columns:not(.is-max-desktop):not(.is-max-widescreen) {
        max-width:1344px
    }
}
@media screen and (min-width: 1408px) {
    .container:not(.is-max-desktop):not(.is-max-widescreen),.section-browse-list .browse-columns:not(.is-max-desktop):not(.is-max-widescreen) {
        max-width:1344px
    }
}
```

**Comment by Monkeychip** [comment]
@infowolfe thank you for the issue. Would you mind including a screenshot to help show the visual effect? You're description is great, but it will help support the problem with a visual.

**Comment by infowolfe** [comment]
@Monkeychip of course!

Here's some screenshots showing this issue in various orientations. Using hard constraints/max-width on the center section of every doc-related thing I've seen from hashicorp makes it pretty unergonomic (especially when trying to read code blocks).

Wide 3840x1080: ![Screen Shot 2024-05-19 at 1 00 15 AM](https://github.com/hashicorp/vault/assets/51030/b1a53e35-a24a-4ea0-b768-f80cfe102f85)
25% corner 1920x1080:  ![Screen Shot 2024-05-19 at 1 00 47 AM](https://github.com/hashicorp/vault/assets/51030/1faf43b1-70b4-4404-91de-a8b8819e6b9c)
Fullscreen ~3840x2160: ![Screen Shot 2024-05-19 at 1 01 34 AM](https://github.com/hashicorp/vault/assets/51030/cd214a18-621b-481b-baca-c1de7bc7625c)
vertical half 1920x2160: ![Screen Shot 2024-05-20 at 12 10 37 PM](https://github.com/hashicorp/vault/assets/51030/8f221aec-ff2a-4adf-a2fe-a34cd7ac335e)
vertical half 1920x2160: ![Screen Shot 2024-05-20 at 12 12 59 PM](https://github.com/hashicorp/vault/assets/51030/54677f1b-72a4-4742-9ac2-d87e5071dd85).

And finally what it looks like if I do a nasty monkeypatch to the stylesheet:
```
@media screen and (min-width: 1408px) {
    .container:not(.is-max-desktop):not(.is-max-widescreen),.section-browse-list .browse-columns:not(.is-max-desktop):not(.is-max-widescreen) {
        max-width: 3840px;
    }
}
```
![Screen Shot 2024-05-20 at 12 20 23 PM](https://github.com/hashicorp/vault/assets/51030/bb41f976-3452-4297-8664-20c67df168b1)

My main concern is that this seems to be an institutional thing across all of Hashicorp's UI/UX teams. It would be really cool if this could be escalated/raised internally so every UI/UX team understands that they need to test on monitors with either hidpi on at 100% and the pages in full screen or on monitors larger than a postage stamp.

**Comment by heatherezell** [comment]
We haven't been able to follow up on this issue due to missing environment details (Vault version, OS, deployment method). Without this information we're unable to reproduce or investigate the report.

If you're still experiencing this, please open a new issue using the bug report template and include full environment details.
